src/components/common/ParentNavTopBar.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { Typography, Menu, Dropdown } from 'antd';
+import { useHistory } from 'react-router-dom';
+import { useOktaAuth } from '@okta/okta-react';
+import { connect } from 'react-redux';
+import { global } from '../../state/actions';
+
+import parent_avatar from '../../assets/icons/parent_avatar.svg';
+
+const { Title } = Typography;
+
+const ParentMenu = props => {
+  const { push } = useHistory();
+  const { authService } = useOktaAuth();
+  const { clearUsers, ...rest } = props;
+
+  const switchUsers = e => {
+    clearUsers();
+    push('/');
+  };
+
+  return (
+    <Menu {...rest}>
+      <Menu.Item key="1" onClick={switchUsers}>
```

Comment:
From what I can see the menu only contains 'Change Users' and 'Log Out'  - I think there could be an iteration of this that also contains 'FAQ' - possibly something to bring up at the next stakeholder meeting? 

Can you think of anything else that would make sense in there?

Reply:
I was thinking the same thing. I will be sure to bring that up in our next standup.